app/models/concerns/authorizable.rb

```diff
@@ -116,5 +108,17 @@ def skip_permission_check
     ensure
       Thread.current[:ignore_permission_check] = original_value
     end
+
+    def find_permission_name(action)
+      type = Permission.resource_name(self.class)
+      permissions = Permission.where(:resource_type => type).where(["#{Permission.table_name}.name LIKE ?", "#{action}_%"])
```

Comment:
One important note is that on some databases but not all, the [underscore represents a single char to look for](https://www.w3schools.com/sql/sql_like.asp) (like the dot in regex), so it might not work well on all databases.

Reply:
I just moved this piece of code from some lines above, so I'd prefer not to do any changes for now as part of this PR. :-)